spec/acceptance/suites/windows/nodesets/default.yml

```diff
@@ -6,30 +6,33 @@
   end
 -%>
 HOSTS:
-  el7:
-    roles:
-      - default
-    platform: el-7-x86_64
-    box: centos/7
-    hypervisor: <%= hypervisor %>
-
   win:
     roles:
       - windows
     platform: windows-server-amd64
-    box: devopsgroup-io/windows_server-2012r2-standard-amd64-nocm
+    box: gusztavvargadr/windows-server
```

Comment:
We replaced an older version of `gusztavvargadr/windows-server` with `devopsgroup-io/windows_server-2012r2-standard-amd64-nocm` in #251.  What's causing the move back?